### ISSUE TITLE
AC: Application Submission Error

### ISSUE THREAD

**Issue by iph-97** [bloom-housing/bloom#3508]
A home seeker in Alameda County reported the following submission error on Depot Apartments:

I am getting "Your application is missing required fields. Please go back and correct this before submitting." error, and does not let me submit my application in
https://housing.acgov.org/applications/review/summary page on the final review.

It does not tell me what is missing,  I went through all my answers, I cannot find anything missing, could you please help me?

This is a high priority issue.

**Comment by sarahlazarich** [comment]
We were unable to repro this and are waiting on more information from the applicant / EIR.

**Comment by ColinBuyck** [comment]
Duplicate of #3487
